backend/src/handlers.rs

```diff
@@ -2,9 +2,9 @@ use crate::{ws, Clients, Result};
 use warp::Reply;
 
 pub async fn ws_handler(ws: warp::ws::Ws, clients: Clients) -> Result<impl Reply> {
```

Comment:
@stephanie-flower the `clients` argument isn't used. Is it important? Also, nowhere in the `ws_handler` function is there a `?` or an `Err`. Do we need to return a `Result`?

Now this is very much more of a personal style thing, so feel free to ignore. It's common in Rust to make a type alias for `Result` to hide the `Err` type, but I don't like it because, what if somewhere else in the code we decide to use `std::io::Result`? Now when I read `Result`, I have to hover over it to find out which it is, or you need to disambiguate it by doing `io::Result`, which sort of defeats the purpose of having shorter type signatures, you might as well write `Result<_, io::Error>`.